### DISCUSSION TITLE
Which commit is used for 6.0.0 build?

### DISCUSSION THREAD

**Discussion by tomvleeuwen** [strongswan/strongswan#2555] [General]
Hi,

Yesterday, the 6.0.0 tag pointed to c86f709b4b2d81fe417a015f0920e251064c417a
Today, the 6.0.0 tag points to bea1f1100ee14264808268bd5cdb6df7a8ea382c

Which one is the actual release and used for the binaries?

Thanks,

Tom

**Comment by strongX509** [comment]
Sorry for the inconvenience. Before the official announcement we added some minor changes at the last minute and moved the 6.0.0 tag point to https://github.com/strongswan/strongswan/commit/bea1f1100ee14264808268bd5cdb6df7a8ea382c. The binaries and the regression test results are based on the new 6.0.0 tag point.

**Reply by tomvleeuwen** [reply]
Thanks, that clarifies it!
